public/cowsay.html

```diff
@@ -0,0 +1,18 @@
+    />
+  </head>
+  <body></body>
+</html>
```

Comment:
what is this HTML? shouldn't be in the PR description?

Reply:
Removed in https://github.com/cowprotocol/cowswap/pull/2439 as I didn't see it being used. @shoom3301 please let me know in case I miss anything here.

Reply:
Thanks for deleting this! It was a test for a link preview in Twitter